### ISSUE TITLE
Focus behavior: Change background-color instead of font-weight

### ISSUE THREAD

**Issue by jhrdina** [PolymerElements/paper-button#95]
Buttons should use background-color change for focus instead of font-weight change.

## Expected outcome

Button's background becomes darker on focus as mentioned in [Google Design Spec](https://www.google.com/design/spec/components/buttons.html#buttons-flat-raised-buttons) and implemented in material-design-lite ([their css](https://github.com/google/material-design-lite/blob/master/src/button/_button.scss)). The font-weight is not changed (default font-weight should be 500 as mentioned in #94).

![after](https://cloud.githubusercontent.com/assets/1280734/12544340/8e3645ee-c339-11e5-8dd3-949cf2cf2034.png)
*Image: material-design-lite [demo](http://www.getmdl.io/templates/dashboard/#), their CSS implementation [here](https://github.com/google/material-design-lite/blob/master/src/button/_button.scss)*

## Actual outcome

Button's font-weight changes on focus, background stays the same.

![before](https://cloud.githubusercontent.com/assets/1280734/12544361/d4666bca-c339-11e5-997a-7256d8374afb.png)
Image: paper-card [demo](https://elements.polymer-project.org/elements/paper-card?view=demo:demo/index.html&active=paper-card)

**Comment by abdonrd** [comment]
:+1:

**Comment by keanulee** [comment]
We decided to use `font-weight` to show focus state instead of background color because we have received feedback that `background-color` may not provide enough contrast for a11y. For this reason, we cannot change the `font-weight` in #94 because that would be too close to the focus state.

**Comment by abdonrd** [comment]
@keanulee oh! :(

Then you should change the spec of MD?
